bin/src/command/orders.rs

```diff
@@ -544,7 +549,7 @@ impl CommandServer {
         }
 
         match received_ok_from_new_process {
-            Some(true) => {
+            Ok(true) => {
```

Comment:
What about using `map` instead and `with_context`, this will reduce the number of line

Reply:
With your comment in mind, I have just added context, but I don't see how `map` helps save lines.